content/images/shared.svg

```diff
@@ -0,0 +1,552 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
The open hand has been effectively deprecated as a sharing symbol.  https://bold.pixelapse.com/minming/share-the-icon-no-one-agrees-on presents some better alternatives.

I agree with his conclusion of the "outgoing tray" icon being the best general-purpose glyph for this.